ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml

```diff
@@ -159,6 +159,16 @@ tests:
         IP_STACK=v4v6
         NETWORK_TYPE=OVNKubernetes
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-ovn-dualstack-local-gateway
+  cron: 0 0 1 1 *
```

Comment:
Why the weird cron? Perhaps `interval` would make more sense?

Reply:
oh yeah, that's my bad. I wanted it 48h. it's updated now. thanks for catching that.